process/resultsParser/returnCode.go

```diff
@@ -0,0 +1,27 @@
+package resultsParser
+
+import (
+	"bytes"
+	"encoding/hex"
+)
+
+// ReturnCode is an enum that describes some common smart contract results.
+type ReturnCode string
+
+const (
+	None       ReturnCode = ""
+	Ok         ReturnCode = "ok"
+	UserError  ReturnCode = "user error"
+	OutOfFunds ReturnCode = "out of funds"
+)
+
+func (rc ReturnCode) String() string {
+	return string(rc)
+}
+
+func fromBuffer(bytes bytes.Buffer) ReturnCode {
```

Comment:
please remove this function, no longer used

Reply:
removed. sorry for that.